### PR TITLE
Update token verification and fix typo

### PR DESCRIPTION
**Pull request by martmull**
- factorize token validation
- fix typo in graphql query generations

### CODE REVIEW

File: server/src/core/api-rest/api-rest-query-builder/factories/find-many-query.factory.ts [hunk 1]
```diff
@@ -6,18 +6,19 @@ import { mapFieldMetadataToGraphqlQuery } from 'src/core/api-rest/api-rest-query
 @Injectable()
 export class FindManyQueryFactory {
   create(objectMetadata, depth?: number): string {
+    const objectNameSingular = capitalize(
+      objectMetadata.objectMetadataItem.nameSingular,
+    );
+    const objectNamePlural = objectMetadata.objectMetadataItem.nameSingular;
```
Comment (charlesBochet) [inline server/src/core/api-rest/api-rest-query-builder/factories/find-many-query.factory.ts:12]: this looks suspicious! also we should capitalize it!

File: server/src/core/api-rest/api-rest-query-builder/factories/find-one-query.factory.ts [hunk 2]
```diff
@@ -6,15 +6,13 @@ import { mapFieldMetadataToGraphqlQuery } from 'src/core/api-rest/api-rest-query
 @Injectable()
 export class FindOneQueryFactory {
   create(objectMetadata, depth?: number): string {
+    const objectNameSingular = objectMetadata.objectMetadataItem.nameSingular;
```
Comment (charlesBochet) [inline server/src/core/api-rest/api-rest-query-builder/factories/find-one-query.factory.ts:9]: I think it should always be capitalized!

**Review by charlesBochet — COMMENTED** [review]
Left comments!

File: server/src/core/api-rest/api-rest-query-builder/factories/find-one-query.factory.ts (hunk 2 above)
Comment (martmull) [reply server/src/core/api-rest/api-rest-query-builder/factories/find-one-query.factory.ts:9]: Nope
image
image

File: server/src/core/api-rest/api-rest-query-builder/factories/find-many-query.factory.ts (hunk 1 above)
Comment (martmull) [reply server/src/core/api-rest/api-rest-query-builder/factories/find-many-query.factory.ts:12]: 🤦‍♂️

File: server/src/core/api-rest/api-rest-query-builder/factories/find-one-query.factory.ts (hunk 2 above)
Comment (martmull) [reply server/src/core/api-rest/api-rest-query-builder/factories/find-one-query.factory.ts:9]: nope 
image

**Review by charlesBochet — APPROVED** [review]
All good, thank you :)

**Pull request merged**